<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MAP_DLG_PrimitiveConfigDialog</class>
 <widget class="QDialog" name="MAP_DLG_PrimitiveConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="m_configWidget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>19</y>
     <width>301</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="m_nameWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>名称</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_nameLineEdit">
         <property name="text">
          <string>放置0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="m_showModeWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>显示模式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_showModeComboBox">
         <item>
          <property name="text">
           <string>map_2d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>map_3d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>map_2d_3d</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="m_modeWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>模式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_modeComboBox">
         <item>
          <property name="text">
           <string>线</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>面</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>墙</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QWidget" name="m_heightWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>高度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_heightComboBox">
         <item>
          <property name="text">
           <string>30000.00</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_heightCheckBox">
         <property name="text">
          <string>贴地</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QWidget" name="m_groupWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>工作组</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_groupLineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QTreeWidget" name="m_treeWidget">
      <column>
       <property name="text">
        <string>属性</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>值</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>放置</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
